### PR TITLE
#1336 Fix ETO forms progress

### PR DESCRIPTION
**Pull request by desfero**
#1336 
localhost_9090_dashboard

### CODE REVIEW

File: app/components/shared/forms/formField/FormTextArea.tsx [hunk 1]
```diff
@@ -75,10 +77,10 @@ export class FormTextArea extends React.Component<FieldGroupProps> {
                   />
                   {suffix && <InputGroupAddon addonType="append">{suffix}</InputGroupAddon>}
                 </InputGroup>
-                <div className="mt-2">
-                  {isNonValid(touched, errors, name) && <div>{errors[name]}</div>}
-                  {charactersLimit && <div>{countedCharacters(value, charactersLimit)}</div>}
-                </div>
+                {isNonValid(touched, errors, name) && (
```
Comment (Moejoe90) [inline app/components/shared/forms/formField/FormTextArea.tsx]: Have you checked how this changes views?

File: app/components/eto/publicView/EtoInvestmentTermsWidget.tsx [hunk 2]
```diff
@@ -70,14 +70,15 @@ export const EtoInvestmentTermsWidget: React.SFC<IProps> = ({
...
             </div>
-            {etoData.discountScheme && (
-              <div className={styles.entry}>
-                <span className={styles.label}>
-                  <FormattedMessage id="eto.public-view.token-terms.price-discount" />
-                </span>
-                <span className={styles.value}>{etoData.discountScheme}</span>
-              </div>
-            )}
+            {/* TODO: It's not possible right now to change `discountScheme` therefore it was removed from interface */}
```
Comment (Moejoe90) [inline app/components/eto/publicView/EtoInvestmentTermsWidget.tsx]: What do you mean by this can you elaborate?

Comment (desfero) [reply app/components/eto/publicView/EtoInvestmentTermsWidget.tsx]: So, I talked with @rudolfix and there is no place on UI to set `discountScheme` right now. We don't that to spend time implementing it at that moment so the quickest way was to remove `discountSheme` from the interface to fix validation on ETO forms.

File: app/components/shared/forms/formField/FormTextArea.tsx (hunk 1 above)
Comment (desfero) [reply app/components/shared/forms/formField/FormTextArea.tsx]: Sure, looks a lot better than the previous one :D
Now:
image
Before:
image

Comment (Moejoe90) [reply app/components/shared/forms/formField/FormTextArea.tsx]: :tada:

File: app/components/eto/publicView/EtoInvestmentTermsWidget.tsx (hunk 2 above)
Comment (Moejoe90) [reply app/components/eto/publicView/EtoInvestmentTermsWidget.tsx]: Ok sounds good

**Pull request merged**